setup.py

```diff
@@ -13,7 +13,6 @@
             cmake_configure_options=[
                 "-DCALL_FROM_SETUP_PY:BOOL=ON",
                 "-DBUILD_PYTHON_BINDINGS:BOOL=ON",
-                f"-DPython3_EXECUTABLE:PATH={sys.executable}",
```

Comment:
In my experience this like is quite important to make `setup.py` (or whichever tool calls it indirectly) to select the same python version that either called `setup.py` or the tool itself. In other words, in a system with multiple python installations, calling either of the following would always pick up the right interpreter:

```
```

Then, the switch to the pybind's logic to find the python interpreter is used when manif in installed with plain CMake. Since also `pybind` kind of vendors a modern `FindPython3`, I think that this option could be read internally. Though, I'm not sure about the `3` in the name.

Reply:
From what I could see, `pybind11` actually does a great job at detecting the appropriate active Python (note that's I'm using it's `PYTHON_EXECUTABLE`). Not only does it work on my machine but also in containers, with and without venv for both. Plus it works on github actions with/without venv too (with/without `actions/setup-python@v2`). 
Plus, it avoids having things too scattered across files.

Reply:
Ok, your tests sound great! My comment was not blocking, the usage of `Python3_EXECUTABLE` could be introduced again in case of problems with the detected version.